migrations/alembic.ini

```diff
@@ -0,0 +1,45 @@
+# A generic, single database configuration.
+
+[alembic]
+# template used to generate migration files
+# file_template = %%(rev)s_%%(slug)s
```

Comment:
Maybe change this to reflect the date and time on the file name. It'll put the migration files in order

Reply:
file_template = %%(year)d%%(month).2d%%(day).2d_%%(hour).2d%%(minute).2d%%(second).2d_%%(slug)s